binding/rust/README.md

```diff
@@ -0,0 +1,110 @@
+# Rhino Speech-to-Intent Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Rhino is Picovoice's Speech-to-Intent engine. It directly infers intent from spoken commands within a given context of
+interest, in real-time. For example, given a spoken command:
+
+> Can I have a small double-shot espresso?
+
+Rhino infers that the user would like to order a drink and emits the following inference result:
+
+```json
+{
+  "isUnderstood": "true",
+  "intent": "orderBeverage",
+  "slots": {
+    "beverage": "espresso",
+    "size": "small",
+    "numberOfShots": "2"
+  }
+}
+```
+
+Rhino is:
+
+* using deep neural networks trained in real-world environments.
+* compact and computationally-efficient, making it perfect for IoT.
+* self-service. Developers and designers can train custom models using [Picovoice Console](https://picovoice.ai/console/).
+
+## Compatibility
+
+- Rust 1.54+
+- Runs on Linux (x86_64), macOS (x86_64), Windows (x86_64), Raspberry Pi, NVIDIA Jetson (Nano), and BeagleBone
+
+## Installation
+First you will need [Rust and Cargo](https://rustup.rs/) installed on your system.
+
+To add the rhino library into your app, add `pv_rhino` to your apps `Cargo.toml` manifest:
+```toml
+[dependencies]
+pv_rhino = "1.6.0"
```

Comment:
maybe say "= "${version}" instead. That way you don't have to update the docs everytime you push a new version